### ISSUE TITLE
New Class for Anchors - Make Jiggly

### ISSUE THREAD

**Issue by NicoleLuxcentric** [luxcentric/luxcentric#37]
I would like 2 anchor options. 
1. To set the font color to "Salmon" (#f16465) DONE = "small-btn"
2. To set Salmon colored DONE = "ext-link"
Make them Jiggly when clicked.

**Comment by gtritchie** [comment]
For "subtle" links to other places on the site, use:
```
<div class="small-btn"><a href="...">My Link</a></div>
```
For links that go outside the site, use `ext-link` instead of `small-btn`. They look the same for now. If you are authoring a blog post you will might want to use `<span>` instead of `<div>`, such as:
```
Here is cool stuff: <span class="ext-link"><a href="https://rstudio.com"  target="_blank" rel="noopener">RStudio</a></span>
```

**Comment by NicoleLuxcentric** [comment]
please make them jiggle when clicked.

**Comment by gtritchie** [comment]
Fixed, they will now move a bit when clicked just like other buttons on the site.

Fixed with 98a8927.

**Comment by NicoleLuxcentric** [comment]
Yay!
